src/main/webapp/common_util.js

```diff
@@ -45,3 +45,22 @@ function clearElement(elementId) {
   }
 }
 
+/**
+ * Diable the button for further click and search.
+ * @param {String} buttonName the name of element of button to disable
+ */
+function disableLoadingButton(buttonName) {
```

Comment:
This name is a bit confusing.

You're not really disabling a "loading" button, your disabling a button during loading.

I think it will be clearest if you change this to `disableButton` and `enableButton` or `disableButtonDuringLoading` and `enableButtonAfterLoading`.

Reply:
Hahah agreed, I think it sounds like you have a button that is called loading.